app/api/resources/metadata.py

```diff
@@ -212,15 +213,20 @@ async def get_genome_ftplinks(request: Request, genome_uuid: str, region_name: s
     "/genome/{genome_uuid}/dataset/{dataset_type}/attributes", name="dataset_attributes"
 )
 async def get_genome_dataset_attributes(
-    request: Request, genome_uuid: str, dataset_type: str
+    request: Request,
+    genome_uuid: str,
+    dataset_type: str,
+    attribute_name: Annotated[list[str] | None, Query()] = None,
```

Comment:
Please add `attribute_name` also to [API spec](https://github.com/Ensembl/ensembl-web-metadata-api/blob/3c2f87215503e0ddfaf223b75dbe570250ed2439/APISpecification.yaml#L238).